thefuck/rules/whois.py

```diff
@@ -1,4 +1,5 @@
-from urllib.parse import urlparse
+ # coding: utf-8
+from urlparse import urlparse
```

Comment:
You made this line work with Python 2, but now it fails with Python 3. The best option is to use [six](https://six.readthedocs.org/). Check #201.